slideview/src/main/java/ng/max/slideview/SlideView.java

```diff
@@ -145,6 +136,9 @@ public void setButtonImageDisabled(Drawable image) {
         buttonImageDisabled = image;
     }
 
+    public void setButtonBackgroundColor(@ColorInt int color) {
```

Comment:
This is not very necessary since we can use the available method which takes a  ColorStateList `setButtonBackgroundColor(ColorStateList color)` by converting  the color with `ColorStateList.valueOf()`

If this must be added, it should only call the overloaded method and pass a converted ColorStateList.